tests/TellerTest.php

```diff
@@ -0,0 +1,412 @@
+<?php
+
+namespace Tests\Money;
+
+use Money\Money;
+use Money\Teller;
+
+class TellerTest extends \PHPUnit\Framework\TestCase
```

Comment:
Let's make this class final. I know people like to debate this, but since it is the library standard, I see no point why this should be different here.